src/graph/test/SchemaTest.cpp

```diff
@@ -27,6 +27,24 @@ class SchemaTest : public TestBase {
     }
 };
 
+TEST_F(SchemaTest, TestComment) {
+    auto client = gEnv->getClient();
+    ASSERT_NE(nullptr, client);
+    // Test command is comment
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "# CREATE TAG TAG1";
+        auto code = client->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::E_STATEMENT_EMTPY, code);
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "SHOW SPACES # show all spaces";
```

Comment:
BTW, add the case :
"-- SHOW SPACES; SHOW SPACES;"
E_STATEMENT_EMTPY

Reply:
It's the same as #